### PR TITLE
Explaing why OpenSSL has been updated

### PR DESCRIPTION
**Pull request by kneth**
@realm/java @bmunkholm

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ### Enhancements
 
-* Upgrading to OpenSSL 1.0.1t (#2749).
+* Upgrading to OpenSSL 1.0.1t. Google Play will on July 11, 2016 only accept apps using OpenSSL 1.0.1r or later (https://support.google.com/faqs/answer/6376725, #2749).
```
Comment (stk1m1) [inline CHANGELOG.md]: `From July 11, 2016  onward, Google Play will only accept apps using OpenSSL 1.0.1r or later version.` if you like to emphasize the date.

**Comment by stk1m1** [comment]
Other than a suggestion, 👍

**Comment by cmelchior** [comment]
👍

**Comment by bmunkholm** [comment]
+1

**Comment by zaki50** [comment]
:+1:

**Pull request merged**